app/models/task.py

```diff
@@ -2,4 +2,25 @@
 
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String, nullable=False)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime)
+    goal_id = db.Column(db.Integer, db.ForeignKey('goal.goal_id'), nullable=True)
+
+    def get_dict(self):
+        is_complete = False
+        if self.completed_at:
+            is_complete = True
+        
+        response_dict =  {
+            'id': self.task_id,
+            'title': self.title,
+            'description': self.description,
+            'is_complete': is_complete
+            }
```

Comment:
You can simplify this using `bool(self.completed_at)`:

```suggestion
        response_dict =  {
            'id': self.task_id,
            'title': self.title,
            'description': self.description,
            'is_complete': bool(self.completed.at)
        }
```